model/version.go

```diff
@@ -38,6 +39,10 @@ type Version struct {
 	BuildVariants       []VersionBuildStatus `bson:"build_variants_status,omitempty" json:"build_variants_status,omitempty"`
 	PeriodicBuildID     string               `bson:"periodic_build_id,omitempty" json:"periodic_build_id,omitempty"`
 
+	// This stores whether or not a version has tasks which were activated.
+	// We use a bool ptr in order to determine whether or not this field was set before it was introduced
```

Comment:
> whether or not this field was set before it was introduced
this sentence reads a little funny. Maybe "to distinguish the unset value from the default value" or something along those lines